### ISSUE TITLE
Vyper code uses meter label

### ISSUE THREAD

**Issue by BenKaufman10** [ethereum/casper#71]
I was going through the code of simple_casper.v.py and saw a strange use of the ``m`` labeling.
In vyper, ``m`` means meter. So why does casper uses it (mostly as ``wei / m``) so much?

**Comment by djrtwo** [comment]
We scale deposits as an optimization. We scale on the way in and the way out, rather than updating all validator deposits at each epoch depending on gains/losses. We use `m` as the unit deposits are scaled against. This extra unit helps with type checking when dealing with wei vs scaled wei

**Comment by djrtwo** [comment]
Correct. Is there a unit currently that might make more sense?

**Comment by BenKaufman10** [comment]
I don't think so, maybe ``currency`` or ``currency1`` will sound more generic but I don't think it will help. This are the current units in Vyper: ``currency, wei, currency1, currency2, sec, m, kg``.
I was thinking about adding some generic unit (e.g. ``unit`` or so) but if we are going to implement VIP 752 it won't be necessary.
